src/schemas/models/AccessToken.yml

```diff
@@ -0,0 +1,10 @@
+type: object
+properties:
+  token:
+    type: string
+  organizationID:
+    type: string
+  role:
+    type: string
```

Comment:
Hier könnten wir auch ein enum verwenden, um es typensicherer zu haben. 💡 

```suggestion
    type: string
    enum:
      - admin
      - editor
      - author
      - member
      - unassigned
```

(Gleiches dann in `User.yml`.)